accelerator/main.c

```diff
@@ -73,8 +73,9 @@ int main(int argc, char* argv[]) {
   }
 
   log_warning(logger_id, "Tangle-accelerator starts running\n");
+
   while (ta_http.running) {
-    ;
+    sleep(5000);
```

Comment:
This is ugly. Can you try `usleep` and/or `sched_yield`?  Instead, we might introduce signals to avoid busy waiting.

Reply:
What about use `pause()` to wait for a signal?

Reply:
If you are willing to use [pause(2)](https://linux.die.net/man/2/pause), its return value `EINTR` shall be taken into consideration carefully.